Add PW7-PW12 Support

jasonacox opened the pull request:
This update adds support for up to 12 Powerwalls (PW1 to PW12) for temperature data, frequencies, voltages and capacity as requested by @anderep in https://github.com/jasonacox/Powerwall-Dashboard/discussions/253 and opened in issue https://github.com/jasonacox/Powerwall-Dashboard/issues/256.

Changs:
* influxdb/influxdb.sql - Added new CQs for PW7-PW12 
* influxdb/run-once-2.9.2.sql - Run once update query to pull old raw data into long term buckets for PW7-PW12
* dashboard.json - Added PW7 to PW12 to graphs (temperature, frequencies, voltages, and capacity)

Request: @mcbirse to review

Closes #256

mcbirse commented on RELEASE.md:
```diff
@@ -1,5 +1,9 @@
+
 ## v2.9.1 - Minimize Grafana Plugins
 
 * Minimize the number of default Grafana plutins as defined in the `grafana.env.sample` file (installed as localized `grafana.env`) for new installations by @jasonacox in https://github.com/jasonacox/Powerwall-Dashboard/issues/234
```
Typo - "plugins" misspelt as "plutins"

jasonacox replied:
😊 Great catch!

mcbirse commented on dashboard.json:
```diff
@@ -7087,7 +7591,795 @@
+              ],
+              "type": "fill"
+            }
+          ],
```
Is PW7-12 L1 & L2 missing? 

i.e. below example for PW7 L1 & L2 (and so on required for all PW7-12). "refId" will need to be adjusted for section.

```json
        {
          "alias": "PW7 L1",
          "datasource": {
            "type": "influxdb",
            "uid": "${DS_INFLUXDB}"
          },
          "groupBy": [
            {
              "params": [
                "$__interval"
              ],
              "type": "time"
            },
            {
              "params": [
                "null"
              ],
              "type": "fill"
            }
          ],
          "hide": false,
          "measurement": "http",
          "orderByTime": "ASC",
          "policy": "vitals",
          "query": "SELECT mean(\"PW7_PINV_VSplit1\") FROM \"vitals\".\"http\" WHERE \"ISLAND_VL1N_Load\" < 200.0 AND  $timeFilter GROUP BY time($__interval) fill(null)",
          "rawQuery": true,
          "refId": "Y",
          "resultFormat": "time_series",
          "select": [
            [
              {
                "params": [
                  "PW7_PINV_VSplit1"
                ],
                "type": "field"
              },
              {
                "params": [],
                "type": "mean"
              }
            ]
          ],
          "tags": []
        },
        {
          "alias": "PW7 L2",
          "datasource": {
            "type": "influxdb",
            "uid": "${DS_INFLUXDB}"
          },
          "groupBy": [
            {
              "params": [
                "$__interval"
              ],
              "type": "time"
            },
            {
              "params": [
                "null"
              ],
              "type": "fill"
            }
          ],
          "hide": false,
          "measurement": "http",
          "orderByTime": "ASC",
          "policy": "vitals",
          "query": "SELECT mean(\"PW7_PINV_VSplit2\") FROM \"vitals\".\"http\" WHERE \"ISLAND_VL1N_Load\" < 200.0 AND $timeFilter GROUP BY time($__interval) fill(null)",
          "rawQuery": true,
          "refId": "Z",
          "resultFormat": "time_series",
          "select": [
            [
              {
                "params": [
                  "PW7_PINV_VSplit2"
                ],
                "type": "field"
              },
              {
                "params": [],
                "type": "mean"
              }
            ]
          ],
          "tags": []
        },
```

jasonacox replied:
Thank you @mcbirse !  Brilliant. 🙇  You are right. I was missing that whole section. I just [commited a change](https://github.com/jasonacox/Powerwall-Dashboard/pull/262/commits/25a465b316ab033f474def97f68671740281b182):
* Added PW7-12 L1/L2
* Updated `params` (even for rawQuery = true) to match
* Re-assigned `refId` to be sequential (A-Z), not required but looks cleaner :) 

If you have time, please take another look before I merge. Thanks for your help with this! 🙏

mcbirse commented on dashboard.json:
```diff
@@ -7087,7 +7591,795 @@
+            [
+              {
+                "params": [
+                  "PW6_PINV_VSplit2"
```
Not sure if this affects anything per se... noticed the field in the "select"  "params" section does not match the field in the query. This occurs throughout in various sections.

jasonacox replied:
I noticed that too. When you switch to "edit query" directly mode, it doesn't update params. It uses the query to render, but if you attempt to switch out of query mode, it will override the selects with the params.  I'll edit.

mcbirse reviewed (changes requested):
A few minor changes may be required. The dashboard.json may be missing PW7-12 L1 & L2 in voltages section? Please refer comments.

New CQ's and run-once SQL all look good.

jasonacox commented:
Thank you @mcbirse !  Brilliant. 🙇  You are right. I was missing that whole section. I just [commited a change](https://github.com/jasonacox/Powerwall-Dashboard/pull/262/commits/25a465b316ab033f474def97f68671740281b182):
* Fixed plugin typo
* Added PW7-12 L1/L2
* Updated `params` (even for rawQuery = true) to match
* Re-assigned `refId` to be sequential (A-Z), not required but looks cleaner :) 

If you have time, please take another look before I merge. Thanks for your help with this! 🙏

mcbirse reviewed (approved):
Updates look good!

jasonacox commented:
Thanks @mcbirse !
